tests/xlsx_export.test.ts

```diff
@@ -759,6 +759,62 @@ describe("Test XLSX export", () => {
       expect(await exportPrettifiedXlsx(model)).toMatchSnapshot();
     });
 
+    test("charts that aggregate labels", async () => {
```

Comment:
I think you can just test `exportForExcel` and assert the charts are not exported.
No need to create (and commit) excel file which contains... nothing [interesting for that test]